chains/53935/0xE05c976d3f045D0E6E7A6f61083d98A15603cF6A/contract-information.json

```diff
@@ -0,0 +1,4 @@
+{
+  "address": "0xE05c976d3f045D0E6E7A6f61083d98A15603cF6A",
```

Comment:
Why does this contract have a logo? I think we should restrict logos to just ERC-20s

Reply:
We were asked to add the logo for this contract (synapse bridge) by product a few weeks ago - is there a reason that we should restrict logos to just ERC-20s?  There is a benefit to adding logos to non-token contracts (such as in this case)